### ISSUE TITLE
credits are offcenter

### ISSUE THREAD

**Issue by Eneocho** [d3athrow/vgstation13#32001]
![offcenter](https://user-images.githubusercontent.com/67024428/152635916-a31d0b2a-6ab8-4c9c-952b-182cbcc6a3b5.png)

Dunno why.

**Comment by DamianX** [comment]
Does this happen all the time?
Looking at that screenshot I'm thinking that Bazoooooooooooooooooooooooooooo may cause this

**Comment by Eneocho** [comment]
Nope, that one round. And I've been in tons of rounds with Bazooooooooooooooooooooooooooo it hasn't happened before, so I assume it's a weird and obscure issue of the restored credits.

**Comment by robintime** [comment]
this still happens.

**Comment by DamianX** [comment]
Consistently?

**Comment by robintime** [comment]
seems like it; i think it's more noticeable in eneocho's screenshot due to the window size, but i notice my credits being off-center as well.
